docs/explanation/charm-maturity.md

```diff
@@ -0,0 +1,88 @@
+
+> See more:
+>  - {external+charmcraft:ref}`How to manage charms | Public a charm <publish-a-charm>`
+
+## Public listing on Charmhub
+
+When an author is satisfied that the charm is suitable for wider production use, they can request public Charmhub listing. The charm will be lightly reviewed to ensure that it does what it says that it does, has suitable documentation, infrastructure, metadata, and so forth. After passing review, the charm will show up in searches on Charmhub, and web searches leading to Charmhub.
+
+> See more:
+>  - [How to request Charmhub public listing](#make-your-charm-discoverable)
+
+Reviewing charms encourages the involvement of the community. The community refers to individuals and organisations creating or contributing to charms, Juju and the wider charming ecosystem.
+
+### Key considerations
+
+The process for making a charm discoverable on Charmhub involves a light review of the charm and its metadata. The goals of that process include:
+
+1. The process is streamlined: The party requesting the review provides structured evidence as input to the review process.
+2. A review is transparent for the community. Review and the review comments are public.
+3. Everyone can participate in the review, for example, participate in a discussion in a GitHub issue. A review may benefit from the expertise of a reviewer in the relevant field. Thus, the review process is flexible and open to involving multiple persons.
+4. The review covers the effective automation of tests for automated approvals of subsequent releases.
+
+### Roles and concepts
+
+|Role or item|Description|
+| --- | --- |
+|Author|Author of the charm or person representing the organisation. The person submitting the charm for review is called the author in this documentation.|
+|Publisher|The responsible person or organisation for publishing the charm.|
+|Review group|A group of contact persons watching for review requests to arrive and requesting modifications or assigning a review to a suitable reviewer. This is currently the Canonical Charm Tech team.|
+|Reviewer|Person conducting the review.|
+|Listing|After the reviewer has reviewed the charm successfully, it can be switched to 'listing'. Listing means that the charm will be part of the search result when querying the web pages and API of Charmhub.io, and is in the Charmhub sitemap (so will be found by third-party search engines). Without 'listing', the charm will be available under its URL but is not listed in searches.|
+
+## Ongoing maintenance and evolution
+
+Public listing is not the end of a charm's growth. Truly mature charms offer more integration across the charming ecosystem (ingress, observability, identity, ...), gracefully handle scaling up and down, and more. These noteworthy charms are most likely to get wide adoption, and be featured in editorial content in the charming world. Some specific standards that Juju users are looking for include:
+
+### The charm has sensible defaults
```

Comment:
All these subsections are covered in the Charm maturity doc, aren't they? I'd just link there. If any bits aren't there yet, I'd incorporate them there.